docs/company.md

```diff
@@ -0,0 +1,335 @@
+      "logo": "/companies/42/logo.png",
+      "website": "magnetis.com.br",
+      "social_links": [
+        "twitter.com/@magnetis"
+      ],
+      "markets": [
+        "financial services",
+        "advisor"
+      ],
+      "locations": [
+        "São Paulo, Brazil"
+      ],
+      "deals": [
+        {
+          "id": "1234",
+          "date": "2015-08-01",
+          "type": "raised funds",
+          "series": "seed",
+          "investors": [
+            "Monashees",
+            "Redpoint e.ventures",
+            "500Startups"
+          ],
+          "currency": "BRL",
+          "amount": 3_000_000
+        }
+      ],
+      "persons": [
+        {
+          "id": "1",
+          "name": "Luciano Tavares",
+          "role": "CEO",
+          "image": "/persons/1/mug.png"
+        },
+        {
+          "id": "2",
+          "name": "Fabiano Beselga",
+          "role": "CTO",
+          "image": "/persons/2/mug.png"
+        }
+      ],
+      "created": {
+        "time": "2012-02-15T15:12:21-05:00",
+        "user_id: 42,
+        "name": "Diego Gomes",
+        "image": "/persons/42/mug.png"
+      },
+      "modified": {
+        "time": "2012-02-15T15:12:21-05:00",
+        "user_id: 48,
+        "name": "Elon Musk",
+        "image": "/persons/48/mug.png"
+      },
+      "verified": {
+        "time": "2012-02-15T15:12:21-05:00",
+        "user_id": "42",
+        "name": "Diego Gomes",
+        "image": "/persons/42/mug.png"
+      }
+    },
+    {
+      "id": "84",
+      "name": "RockContent",
+      "description": "RockContent rocks!",
+      "logo": "/companies/84/logo.png",
+      "website": "rockcontent.com.br",
+      "social_links": [
+        "twitter.com/@rockcontent"
+      ],
+      "markets": [
+        "content marketing"
+      ],
+      "locations": [
+        "Belo Horizonte, Brazil"
+      ],
+      "deals": [
+        {
+          "id": "2345",
+          "date": "2014-08-01",
+          "type": "raised funds",
+          "series": "seed",
+          "investors": [
+            "Napkn Ventures"
+          ],
+          "currency": "BRL",
+          "amount": 300_000
+        }
+      ],
+      "persons": [
+        {
+          "id": "34",
+          "name": "Edmar Ferreira",
+          "role": "CEO",
+          "image": "/persons/34/mug.png"
+        }
+      ],
+      "created": {
+        "time": "2012-02-15T15:12:21-05:00",
+        "user_id: 42,
+        "name": "Diego Gomes",
+        "image": "/persons/42/mug.png"
+      },
+      "modified": {
+        "time": "2012-02-15T15:12:21-05:00",
+        "user_id: 48,
+        "name": "Elon Musk",
+        "image": "/persons/48/mug.png"
+      },
+      "verified": {
+        "time": "2012-02-15T15:12:21-05:00",
+        "user_id": "42",
+        "name": "Diego Gomes",
+        "image": "/persons/42/mug.png"
+      }
+    }
+  ]
+}
+````
+
+===
+
+`GET /1/companies/{{company_id}}.json`
```

Comment:
I don't have a strong opinion about this. This version is a pattern in Rails, but in Phoenix the APIs don't have the `.json` in it (we can add if we want). 

From one side, it's possible to have more formats for the API adding `.json` to it (XML? 😬 ). On the other side, we know that we don't want to serve another format, and it's implicit that this is JSON, as @rafaelrinaldi said.

I'm more inclined to remove the `.json` too.